test/hotspot/jtreg/serviceability/jvmti/ObjectMonitorUsage/ObjectMonitorUsage.java

```diff
@@ -28,18 +28,17 @@
  * @summary converted from VM Testbase nsk/jvmti/GetObjectMonitorUsage/objmonusage003
  * DESCRIPTION
  *     The test checks if the JVMTI function GetObjectMonitorUsage returns
- *     the expected values for the owner, entry_count, water_count
+ *     the expected values for the owner, entry_count, waiter_count
  *     fields of JVMTI_monitor_info.
  *     The testcases are the following:
- *       - unowned object without any waitings
- *       - unowned object with waitings to be notified
- *       - owned object without any waitings
- *       - owned object with N waitings to enter the monitor
- *       - owned object with N waitings to be notified
- *       - owned object with N waitings to enter, from 0 to N waitings to re-enter,
- *         from N to 0 waitings to be notified
+ *       - unowned object without any waiting threads
+ *       - unowned object with threads waiting to be notified
+ *       - owned object without any waiting threads
```

Comment:
You could say "threads waiting" in all cases - it looks odd to reverse it for two cases.

Reply:
Thanks, updated.